dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -699,9 +699,7 @@ public WorkflowVersion getTagVersion(GHRepository ghRepository, String gitRefere
         // Find existing version if it exists
         Optional<WorkflowVersion> existingVersion = workflow.getWorkflowVersions().stream().filter(workflowVersion -> Objects.equals(workflowVersion.getReference(), gitReference)).findFirst();
         Map<String, WorkflowVersion> existingDefaults = new HashMap<>();
-        if (existingVersion.isPresent()) {
-            existingDefaults.put(gitReference, existingVersion.get());
-        }
+        existingVersion.ifPresent(workflowVersion -> existingDefaults.put(gitReference, workflowVersion));
```

Comment:
👍 I'd been meaning to look for something like this but had never bothered; off-hand this seems a lot nicer than `if (obj.isPresent()) {...`

Reply:
This pattern, oft-repeated, is also picked up by automated code cleanup :)

https://github.com/dockstore/dockstore/pull/2491/files#diff-bad96ec8d55631306683aaeca457ef99L615

Reply:
That PR has been open for a while, you might want to run a few of those patterns in a new PR (or several smaller, shorter lived PRs) from develop to avoid the merge conflicts.